content/source/docs/enterprise/private/migrate.html.md

```diff
@@ -9,6 +9,24 @@ sidebar_current: "docs-enterprise2-private-installer-migration"
 This document outlines the procedure for migrating from the AMI-based Private Terraform Enterprise (PTFE)
 to the Installer-based PTFE.
 
+## Backup
+
+Before beginning, it's best create an additional backup of your RDS database. This will allow you to rollback the data and continue to use the AMI if necessary.
+
+To do so, go to the [Amazon RDS Instances](https://console.aws.amazon.com/rds/home?region=us-east-1#dbinstances:). You may need to change the region that you are viewing in order to see your PTFE instance. Once you find it, click on the instance name. On the next page, select **Instance Actions** and then **Take snapshot**. On the **Take DB Snapshot** page, enter a name for the snapshot such as _pre installer migration_, and then click **Take Snapshot**.
+
+The snapshot will take a little while to create. After it has finished, you can continue with the rest of the migration steps.
+
+### Reverting back to AMI
+
+If you wish to revert back to the AMI after running the migration script, first if you've already manipulated the state file to move the resources, you'll need to restore the original state file.
```

Comment:
~If you wish~ To revert ~back~ to...

I think we'll want to separate and number these steps, so this would go:

> ....after running the migration script:

>1. If you've already manipulated the state file to move the AMI resources, restore the original state file.
>2. With the state file in place, return...
>3. Run `terraform apply`...
>4. Return to the original hostname used for the cluster.

Reply:
@koikonom 's note about backing up the state file might fit well in item 1 here, too?